src/service.ts

```diff
@@ -6,6 +6,7 @@ import { mount } from './database';
 import { findMailboxId } from './findMailboxId';
 import { createHelpScoutClient } from './helpScoutClient';
 import { formatText } from './helpScoutTemplate';
+import {encodeHtml} from './htmlEntities'
```

Comment:
```suggestion
import { encodeHtml } from './htmlEntities';
```

prettier without eslint makes you lazy